queries.sql

```diff
@@ -14,3 +14,15 @@ SELECT animals.name, animals.neutered FROM animals WHERE neutered = true;
 SELECT animals.name FROM animals WHERE name != 'Gabumon';
 SELECT animals.name, animals.weight_kg FROM animals 
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- 
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT ROUND(AVG(weight_kg),4) "Average weight" FROM animals;
+SELECT * FROM animals WHERE escape_attempts > 0 AND neutered = true;
```

Comment:
* Kindly use the right query here for `Who escapes the most, neutered or not neutered animals?` your query can be narrowed down by using the `Group by neutered` keyword this would narrow the search to just animals who were neutered whether `true` or `false`